ecommerce_extensions/payment/processors/edupay.py

```diff
@@ -179,6 +179,7 @@ def get_transaction_parameters(self, basket, **kwargs):  # pylint: disable=unuse
         parameters.update(self._get_owner_parameters(basket.owner))
         parameters.update(self.ednx_configuration.get('extra_parameters', {}))
         single_seat = self.get_single_seat(basket)
+        parameters['course_name'] = single_seat.course.name
 
         if single_seat:
             parameters['description'] = single_seat.course_id
```

Comment:
why don't you move it to this line? otherwise it will raise errors when signle_seat does not exist

Reply:
is still working OK after moving it?